adr/ADR-32.md

```diff
@@ -223,10 +242,6 @@ The type for this is `io.nats.micro.v1.schema_response`.
     * Average processing_time is the total processing_time divided by the num_requests
     */
     average_processing_time: Nanos;
-    /**
-    * ISO Date string when the service started
-    */
-    started: string
```

Comment:
just verifying that this is to be removed

Reply:
I was thinking about this and if we want to configure endpoints together with the service, it may make more sense to have `started` on the top level (for the whole service, not particular endpoints).

Reply:
That is what I did already on my PR - started is a common info.